.circleci/build.sh

```diff
@@ -33,6 +33,9 @@ pip install ninja
 # Install the Lark parser required for the XLA->ATEN Type code generation.
 pip install lark-parser
 
+# Install hypothesis, required for running some PyTorch test suites
+pip install hypothesis
```

Comment:
Hmmm, this means we have to install as well.
Would it be possible to not add extra module deps here?

Reply:
Where would you suggest we install hypothesis?

Reply:
Where is it used?

Reply:
It's imported by test_nn.py

Reply:
Then it will need to be installed from here as well:

https://github.com/pytorch/xla/blob/master/scripts/build_torch_wheels.sh

@jysohn23 Any other places?

Reply:
Our GCE Image creation workflow. I'll add that.

Reply:
@jysohn23 do you want to add it in this PR? I can also toss the line in.

Reply:
(I assume your approval means not in this PR)